src/runners/ArgumentParser.ts

```diff
@@ -0,0 +1,79 @@
+import * as yargs from "yargs";
+import * as fs from "fs";
+
+/**
+ * Ensures json is stored as string in parameters. If file is provided, function transforms it to string.
+ * @param argv returns modified arguments, where test parameters are store in string json
+ */
+function parametersParser(argv: any) {
+	// check if test parameters was provided
+
+	const testParameters = argv.test_parameters as string;
+
+	if (testParameters !== undefined) {
+		let jsonRawText = null;
+
+		if (testParameters.endsWith(".json")) {
+			// it is file, read it and parse it
+			jsonRawText = fs.readFileSync(testParameters, { encoding: "utf8" });
+		}
+		else {
+			jsonRawText = testParameters;
+		}
+
+		argv.test_parameters = jsonRawText;
+	}
+	return argv;
+}
+
+/**
+ * Parse args passed to process.
+ * @author Marian Lorinc <mlorinc@redhat.com>
+ */
+export default function parseArgs(): { [key: string]: any } {
```

Comment:
can we move export of function to end of the file to keep it same as in all other .ts files or not?